test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -1827,6 +1827,9 @@ def convert_to_dictlist(self, l, attribs=None, mergelines=True, id=None):
                     _e = len(lines) - 1
                     lines[_e] = lines[_e].strip('\r\n\t') + \
                         l[i].strip('\r\n\t')
+                elif not l[i].startswith(('\t', ' ')) and l[i].endswith(','):
+                    _e = len(lines) - 1
+                    lines[_e] = lines[_e] + l[i]
```

Comment:
@dinesh1306 - this issue will be seen whenever we will have a env var set to a multi line value. Please try testing with another multi-line env var and see if the fix works